### DISCUSSION TITLE
Re-Link commits with terminated student email to account

### DISCUSSION THREAD

**Discussion by KSneijders** [community/community#36337] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey all,

Today I was cleaning up some accounts including GitHub. 
I saw my old student email as one of the listed emails and removed it without a second thought. 

Now I just noticed that all commits associated with this email are no longer linked to my account. 
I understand why this happened but I wasn't able to find a way to undo it. 

I **cannot** add the email back as it has been terminated when I graduated. 
I also saw that at least in 2020, GitHub doesn't support `mailmap` files: [link](https://github.com/orgs/community/discussions/22518).

The same post says I could rewrite git history which is an option but I'd prefer to just link that email back to my account one way or the other if possible. So, is that possible in any way?

Thanks in advance :)

**Comment by loganbek** [comment]
Pretty sure you're going to have to verify the email so if you don't have access to it anymore I don't see a way unfortunately.

**Comment by siddhpant** [comment, marked as answer]
Just add the email. Your account will show up even if the email is unverified. Idk why that is the case, but okay.

Yes, it's a hack, but is the only solution in this case.

**Reply by KSneijders** [reply]
Wow, you're right - that's very interesting. Hope they don't fix it without an alternative way to change it. Thanks! ❤️ 

PS: Took a couple days to respond as it takes a while for the change to take effect.

**Comment by ossanna16** [comment]
Hey there @KSneijders and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by Tchisom17** [comment]
Hey everyone,

I was using a work mail for my git which was not linked to my github account, and upon adding it I couldn't find the contributions I did with the email on my graph.

How do I resolve this?

**Reply by KSneijders** [reply]
How long ago was this? It takes a couple days to update
